test-app/build.gradle

```diff
@@ -20,9 +20,10 @@ android {
 
 repositories {
     google()
+    mavenCentral()
     jcenter()
 }
 
 dependencies {
-    androidTestCompile 'com.android.support.test.espresso:espresso-core:2.2.2'
+    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
```

Comment:
I guess we have not completed AndroidX migration

Reply:
Was checking if we could use `androidx.test.espresso:espresso-core:3.1.0'`

Reply:
Nope. Probably makes sense to do that when we remove the junit 3 tests, I think.